workspaces/kiali/plugins/kiali-backend/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@backstage-community/plugin-kiali-backend",
+  "version": "1.19.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "backend-plugin",
+    "supported-versions": "1.31.3",
+    "pluginId": "kiali",
+    "pluginPackages": [
+      "@backstage-community/plugin-kiali",
+      "@backstage-community/plugin-kiali-backend"
```

Comment:
@aljesusg Was able to fix the this issue on my side by removing `"@backstage-community/plugin-kiali"` from here: 
```suggestion
      "@backstage-community/plugin-kiali-backend"
```